Flutter/pdf/working-with-annotations.md

```diff
@@ -680,4 +680,114 @@ File('output.pdf').writeAsBytesSync(await document.save());
 document.dispose();
 
 {% endhighlight %}
- 
\ No newline at end of file
+ 
+## Importing annotations from FDF file
+
+FDF stands for Forms Data Format. FDF is a file format for representing annotations present in a PDF document. You can import annotation data from the FDF file to PDF using the importAnnotation method in PdfDocument class.
+
+{% highlight dart %}
+
+    //Load an existing PDF document.
+    PdfDocument document =
+        PdfDocument(inputBytes: File('input.pdf').readAsBytesSync());
+    //Import the annotations to FDF file format.
+    document.importAnnotation(
+        File('import.fdf').readAsBytesSync(), PdfAnnotationDataFormat.fdf);
+    //Save the PDF document.
+    File('output.pdf').writeAsBytesSync(await document.save());
+    //Dispose the document.
+  document.dispose();
+
+{% endhighlight %}
+
+## Importing annotations from XFDF file
+
+XFDF stands for XML Forms Data Format. XFDF is the XML version of FDF for representing annotations that are contained in a PDF document. You can import annotation data from the XFDF file to PDF using the importAnnotation method in PdfDocument class.
+
+{% highlight dart %}
+
+    //Load an existing PDF document.
+    PdfDocument document =
+        PdfDocument(inputBytes: File('input.pdf').readAsBytesSync());
+    //Import the annotations to XFDF file format.
+    document.importAnnotation(
+        File('import.xfdf').readAsBytesSync(), PdfAnnotationDataFormat.xfdf);
+    //Save the PDF document.
+    File('output.pdf').writeAsBytesSync(await document.save());
+    //Dispose the document.
+    document.dispose();
+
+{% endhighlight %}
+
+## Importing annotations from JSON file
+
+JSON stands for JavaScript Object Notation. It is a collection of key or value pairs and it is used for serializing and transmitting the structured data over a network connection. You can import the annotation data from the JSON file to PDF using the importAnnotation method in PdfDocument class.
+
+{% highlight dart %}
+
+    //Load an existing PDF document.
+    PdfDocument document =
+        PdfDocument(inputBytes: File('input.pdf').readAsBytesSync());
+    //Import the annotations to JSON file format.
+    document.importAnnotation(
+        File('import.json').readAsBytesSync(), PdfAnnotationDataFormat.json);
+    //Save the PDF document.
+    File('output.pdf').writeAsBytesSync(await document.save());
+    //Dispose the document.
+    document.dispose();
+
+{% endhighlight %}
+
+## Exporting annotations to FDF file
+
+To export annotation data to the FDF file from PDF document, you can use the exportAnnotation method in PdfDocument class.
```

Comment:
To export annotation data to the FDF file from the PDF document, you can use the exportAnnotation method in PdfDocument class.